apps/web/src/lang/fr.json

```diff
@@ -10434,6 +10434,10 @@
     "defaultMessage": "de toute durée. (à court terme, à long terme ou indéterminée)",
     "description": "Label displayed on Work Preferences form for any duration option"
   },
+  "uIUSdN": {
+    "defaultMessage": "Apprenez-en davantage sur les modalités du site GC Talents numériques.",
```

Comment:
GC Digital Talent has been translated in French the past not as GC Talents numériques but as Talents numériques du GC. Has something changed?

Reply:
Fixed by [57d4cf5](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9143/commits/57d4cf504103cd58c410c7de27c2010e55b0a3ae)